dgds_backend/config_data/datasets.json

```diff
@@ -25,6 +25,14 @@
       "units": "m/s",
       "timeSpan": "Live",
       "toolTip": "Water level, storm surge, tide and current forecasts from the Global Storm Surge Information System (GLOSSIS) at Deltares. This includes 10 day forecasts at hundreds of nearshore locations across the world. See https://www.deltares.nl/en/projects/global-storm-surge-information-system-glossis for more information.",
+      "flowmapLayer":  {
+        "url": "https://storage.googleapis.com/dgds-flowmap/tiles/glossis-currents-latest/{z}/{x}/{y}.png",
```

Comment:
We're only enabling flowmap for the latest timestamp?

Reply:
no, but I do not know yet how to add  more timestamps

Reply:
Adding timesteps  is now a separate issue